daemon/mgr/cri_test.go

```diff
@@ -1,11 +1,13 @@
 package mgr
 
 import (
+	"strings"
 	"testing"
 
 	apitypes "github.com/alibaba/pouch/apis/types"
 	"github.com/stretchr/testify/assert"
 
+	"fmt"
```

Comment:
Please move this pkg to the top, see: 

> When importing packages, to improve readabilities, we should import package by sequence: system packages, project's own packages and third-party packages. And we should keep a blank line among these three kinds of packages;

in https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md